### ISSUE TITLE
locale support(!) 

### ISSUE THREAD

**Issue by tirolel** [Mottie/Keyboard#783]
Hi @Mottie.

Iam wondering if the keyboard does allow this scenario.

Iam using numbro to support locale options that can be used on my webapp.
I have already defined some languages(decimal & thousand seperators) in numbro.

Is there anyway I can specify an option to keyboard about what is my decimal & thousand seperators? So when I show a custom num layout, it should validate the number regarding the defined option?

E.g.
A user with from Belgium has these options:
delimiters: {
       thousands: " ",
       decimal: ","
}

When the user clicks an input field, it should show the numeric keyboard with the given "," decimal seperator & and if the user enters "1000.00" , it should be like "1 000,00". Also its important to user can add only single decimal.

Can this be done?

Thanks in advance!
Best regards.

**Comment by Mottie** [comment]
Hi @tirolel!

I guess it's not super-obvious, or documented that well, but the [`display` option](https://github.com/Mottie/Keyboard/wiki/Language#display) has a `dec` setting ([demo](https://jsfiddle.net/Mottie/egb3a1sk/7181/))

```js
display: {
  // Decimal point for num pad (optional), change '.' to ',' for European format
  dec: '.:Decimal'
}
```

- The `.` to the left of the colon sets the decimal value for the keyboard.
- The `Decimal` text to the right of the colon sets the keyboard key title (seen on hover)

So for a Belgium keyboard, you can use `dec: ',:Decimal'`. I did try to include the common formatting in the language files (e.g. [German](https://github.com/Mottie/Keyboard/blob/master/languages/de.js)), but not all languages have corresponding files.

The `{dec}` key is set up to only allow the user to type in that decimal character once. You just have to be cautious not to also include the comma key in the layout as it isn't disabled.

**Comment by tirolel** [comment]
Thank you @Mottie for your fast response.

Yes I see and I just have tested the decimal option, and works flawlessly!
The reason is, I do have a screen is very small(480x270) and I made custom layouts for each language(tough work)

Also I have a question about a feature(s).
Can i add my own custom button? 
   - normal button with custom icon & custom functionality?
   - wide button with custom icon & custom functionality?

**Comment by Mottie** [comment]
Yes, they are called [action keys](https://github.com/Mottie/Keyboard/wiki/Actions).

**Comment by tirolel** [comment]
@Mottie Thank you again!
I have a stange problem. not sure if I should start new issue about it or just ask it here.
When I add a keyboard to element, and I enter the input, it doesnt show up a keyboard always. 
If I do refresh few times and try again sometimes it work sometimes it doesnt.
I found out if I give some time it will "initialize" the library and shows up.
Is there anyway I can wait for the keyboard-library to fully initialized? 
The webapp running on rp3.
